pyxem/utils/expt_utils.py

```diff
@@ -606,7 +606,7 @@ def find_beam_center_blur(z, sigma):
     return dispatcher.array(center)
 
 
-def find_beam_offset_cross_correlation(z, radius_start, radius_finish):
+def find_beam_offset_cross_correlation(z, radius_start, radius_finish, **kwargs):
```

Comment:
It's important to inform where the keyword arguments end up. An entry should be added to `Parameters`:

```python
    Parameters
    ----------
    radius_start : int
        The lower bound for the radius of the central disc to be used in the
        alignment.
    radius_finish : int
        The upper bounds for the radius of the central disc to be used in the
        alignment.
    **kwargs
        Keyword arguments passed to :func:`skimage.registration.phase_cross_correlation`.
```

Reply:
Might as well describe the `z` parameter as well while you're at it!